bigtable/metricscaler/metricscaler.py

```diff
@@ -140,16 +143,16 @@ def main(
         'bigtable_instance',
         help='ID of the Cloud Bigtable instance to connect to.')
     parser.add_argument(
-        'bigtable_cluster',
-        help='ID of the Cloud Bigtable cluster to connect to.')
+        'bigtable_clusters',
+        help='ID of the Cloud Bigtable clusters to connect to.')
     parser.add_argument(
         '--high_cpu_threshold',
         help='If Cloud Bigtable CPU usage is above this threshold, scale up',
-        default=0.6)
+        default=0.4)
```

Comment:
Why are these defaults changing?

Reply:
With replication mode CPU recommended 40%, but default can be 0.6

Reply:
oh, could you comment that somewhere? Or just link to the replication docs in a comment where they explain more about the parameters

Reply:
https://cloud.google.com/bigtable/docs/monitoring-instance